apps/alpha-test-app/src/components/layout-demo/LayoutSettings.vue

```diff
@@ -0,0 +1,165 @@
+<script lang="ts" setup>
+import { computed, ref } from "vue";
+import SingleSettingsGroup from "./SingleSettingsGroup.vue";
+import MultiSettingsGroup from "./MultiSettingsGroup.vue";
+import { watch } from "vue";
```

Comment:
import can be combined with the first one from "vue"

Reply:
weird, in the past this happened automatically :O